replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "Replication Report"
+author: "Yari & Kyonne" 
+date: "April 26, 2015"
+output: html_document
+---
+
```

Comment:
please add a bit of description at the beginning so that we can understand about your the context and goals of the replication